shell/shared/grain.js

```diff
@@ -375,11 +375,23 @@ if (Meteor.isClient) {
     },
   });
 
+  Template.grainBackupButton.onCreated(function () {
+    this.isLoading = new ReactiveVar(false);
+  });
+
+  Template.grainBackupButton.helpers({
+    isLoading: function () {
+      return Template.instance().isLoading.get();
+    }
+  });
+
   Template.grainBackupButton.events({
-    "click button": function (event) {
+    "click button": function (event, template) {
+      template.isLoading.set(true);
```

Comment:
Perhaps you also want to prevent subsequent clicks from triggering further `backupGrain` calls here too?